vrt/tests.vrt.js

```diff
@@ -20,8 +20,22 @@ const toMatchImageSnapshot = configureToMatchImageSnapshot({
 
 expect.extend({toMatchImageSnapshot});
 
+const allScenarios = [
+  ...getAllScenarioNames().map(scenarioName => ({
+    scenarioName,
+    theme: 'light',
+  })),
+  // remove rtl from dark theme checks
+  ...getAllScenarioNames()
+    .filter(scenarioName => !scenarioName.includes('rtl'))
+    .map(scenarioName => ({
+      scenarioName,
+      theme: 'dark',
+    })),
```

Comment:
why have the dark tests part of the array as opposed to also being directly in the forEach? i think it may be a little easier to read if all the "passes" are explicitly in the forEach. kind of like this:

```js
getAllScenarioNames().forEach(scenarioName => {
  const config = getConfigFromScenarioName(scenarioName);

  it() // scenarioName-snap.png

  it() // scenarioName__interactionName-snap.png

  it() // scenarioName--dark-snap.png

  it() // scenarioName--mobile-snap.png
});
```

not a huge deal either way